dbt_project.yml

```diff
@@ -42,6 +42,8 @@ vars:
     snowplow__ga4_categories_seed: snowplow_unified_dim_ga4_source_categories
     snowplow__geo_mapping_seed: snowplow_unified_dim_geo_country_mapping
     snowplow__rfc_5646_seed: snowplow_unified_dim_rfc_5646_language_mapping
+    snowplow__grant_select_list: []
+    snowplow__grant_schemas: true
```

Comment:
I know it would be longer but wouldn't `snowplow__grant_usage_on_schemas` be more explicit and therefore potentially less confusing? Same with `snowplow__grant_select_on_tables`? The [] should be self explanatory it is a list anyway.

Reply:
My only concern is those make it sound like the value should be the schemas and tables you want to grant on?